lib/src/css_class_set.dart

```diff
@@ -327,4 +328,36 @@ abstract class CssClassSetImpl implements CssClassSet {
    * This is intended to be overridden by specific implementations.
    */
   void writeClasses(Set<String> s);
+
```

Comment:
I think you're missing the signature change to singleWhere.

Reply:
done.